wagtail/admin/widgets/auto_height_text.py

```diff
@@ -5,11 +5,14 @@
 
 
 class AdminAutoHeightTextInput(widgets.Textarea):
-    template_name = "wagtailadmin/widgets/auto_height_text_input.html"
 
     def __init__(self, attrs=None):
         # Use more appropriate rows default, given autoheight will alter this anyway
-        default_attrs = {"rows": "1"}
+        default_attrs = {
+            "rows": 1,
+            "data-controller": "w-autosize",
+            "data-w-autosize-target": "textarea",
```

Comment:
```suggestion
```

You only need to use targets if you are wanting to target other nodes within the controller scope. If you just need the controlled element, that works for 'free' with stimulus via `this.element`. 

See https://stimulus.hotwired.dev/reference/controllers#properties